BUILD.bazel

```diff
@@ -3934,14 +3941,17 @@ ALL_NEONDOT_MICROKERNEL_SRCS = [
     "src/qu8-gemm/gen/8x8c4-minmax-rndnu-neondot.c",
     "src/qu8-gemm/gen/8x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/1x8c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/1x16c4-minmax-fp32-neondot.c",
     "src/qu8-igemm/gen/1x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/1x32c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/2x8c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/2x16c4-minmax-fp32-neondot.c",
     "src/qu8-igemm/gen/2x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/2x32c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x8c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x32c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/4x8c4-minmax-fp32-neondot.c",
```

Comment:
Note: this doesn't match the GEMM microkernel (which is `4x16c4`)

Reply:
ah, yet another Basel typo :\